packages/system-tests/assets/sfdx-simple/.vscode/settings.json

```diff
@@ -1,12 +1,13 @@
 {
   "extensions.autoUpdate": false,
-  "terminal.integrated.env.osx": {
+"terminal.integrated.env.osx": {
```

Comment:
This file should probably not be committed. @Sweetman, is there a reason that you committed this file? I thought this was supposed to be a pristine workspace so we don't want any .vsode/settings.json when we run the tests.